yarn-project/end-to-end/src/spartan/transfer.test.ts

```diff
@@ -99,43 +96,6 @@ describe('token transfer test', () => {
       wallets.map(w => tokenAdminWallet.methods.mint_public(w.getAddress(), MINT_AMOUNT).send().wait({ timeout: 600 })),
     );
 
-    logger.verbose(`Minting ${MINT_AMOUNT} private assets to the ${wallets.length} wallets...`);
```

Comment:
Feels odd to not run the private transfers at all, but it satisfies the criteria for the test, and makes it simpler to run because you avoid some of your issues 🤷